modules/retail/pages/retail-deploy-terminals.adoc

```diff
@@ -120,7 +128,64 @@ image::retail_imagesync_formula.png[scaledwidth=80%]
 
 == Deploy Images to the Terminals
 
+When you have your bootstrap image ready, you can deploy the image to the terminals.
+
+
+
 .Procedure: Deploying Images to the Terminals
 
 . Power on your POS terminals.
 . The branch server will bootstrap the terminals according to the data you have provided.
+
+
+
+== Deploy Terminals with a Removable USB Device
+
+If you do not want to boot terminals from the network, you can create a live USB image and deploy terminals using a removable USB storage device.
+This is useful if you are unable or unwilling to connect some or all of your terminals to the network, or if you do not have a local {smr} branch server providing network services.
+
+You can prepare a bootable USB device with the image and tools required to deploy a POS terminal using a remote {smr} branch server.
+POS devices booted using the USB device are assigned to the {smr} branch server that created the USB device.
+
+You can create the bootable USB device on the branch server directly, or on the {smr} Server.
+
+
+
+.Procedure: Creating a Bootable USB Device on the Branch Server
+. On the {smr} branch server, at the command prompt, as root, create the POS image:
++
+----
+salt-call image_sync_usb.create <usb image name> <size in MiB>
+----
+. Insert the USB device into the {smr} branch server machine, and copy the image to the new location:
++
+----
+dd if=<usb image name> of=<path to usb device>
+----
+
+
+
+.Procedure: Creating a Bootable USB Device on the {smr} Server
+. On the {smr} Server, at the command prompt, as root, create the POS image:
++
+----
+salt <branch_server> image_sync_usb.create <usb image name> <size in MiB>
+----
+. Insert the USB device into the {smr} Server machine, and copy the image to the new location:
++
+----
+salt <branch_server> cmd.run "dd if=<usb image name> of=<path to usb device on branch>"
+----
```

Comment:
This is not correct. The snipped was meant to show that user can trigger usb stick build from server, however the resulting image is still on branch server.
So unless user transfers the image from branch server to suma server, usb device needs to be available on branch server.
I think this section brings only more confusion and probably best is to remove this procedure and leave only the first one where the image is created on branch server.